### PR TITLE
[IMP] code_survey: Ordering the code question by the code execution t…

### PR DESCRIPTION
**Pull request by daav-odoo**
…ime.

### Description
Sorting the tables of code questions by the execution time.

Link to task: [2856501](https://www.odoo.com/web#model=project.task&id=2856501)

### All Submissions:

* [x] My commit respects the [Odoo commit guideline](https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#git)
* [x] My commit message respects the [commit template](https://github.com/odoo-ps/psbe-process/wiki/Commits-message-guidelines#template)
* [ ] I have used pre-commit
* [x] The PR contains **only** my modification and **no other external** commit

### Sh/Runbot:

* [ ] The commits pass test and the branch is green
* [ ] Unit tests have been implemented / standard ones rewritten
* [ ] The Staging is ISO-Prod and will contain only this dev

### Upgrade:

* [ ] The data affected (*if any*) by the changes has been migrated

### CODE REVIEW

File: code_survey/models/survey_user_input.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import queue
```
Comment (yall-odoo) [inline code_survey/models/survey_user_input.py:1]: no borrar esto

Comment (yall-odoo) [reply code_survey/models/survey_user_input.py:1]: lo necesito para las colas del multiproc

File: code_survey/models/survey_user_input.py [hunk 2]
```diff
@@ -203,8 +203,14 @@ def save_lines(self, question, answer, comment=None):
...
                 else:
-                    self._run_tests_with_interruption(question, solution, test_dict)   
-                    self._save_line_code_answer(question, self.question_score, answer)
+                    startTesting = time.time()
+                    # print("Start: ",startTesting)
+                    self._run_tests_with_interruption(question, solution, test_dict)
+                    endTesting = time.time()
+                    # print("End: ",endTesting)
+                    time_execution = "{0:.3f}".format((endTesting - startTesting))
+                    # print("TimePerQuestion: ",type(time_execution))
```
Comment (yall-odoo) [inline code_survey/models/survey_user_input.py]: borrar prints

**Review by yall-odoo — CHANGES_REQUESTED** [review]
Formato y algunos prints y yasta

File: code_survey/models/survey_user_input.py (hunk 1 above)
Comment (daav-odoo) [reply code_survey/models/survey_user_input.py:1]: Me dice ""queue" is not accessed Pylance".

Comment (yall-odoo) [reply code_survey/models/survey_user_input.py:1]: Lo tengo en mi repo local el uso de esta libreria

Comment (yall-odoo) [reply code_survey/models/survey_user_input.py:1]: Bueno no importa. Lo agregaré después

**Pull request merged**